labels/de-DE.json

```diff
@@ -689,6 +689,7 @@
   "subscriptionItemAddedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
   "subscriptionItemQuantityUpdatedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
   "subscriptionItemRemovedBlob": "Hallo, <br/><br/> Vielen Dank für Ihren Einkauf bei uns unter {0}. Ihr Abonnement wurde aktualisiert. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, bitte besuche Mein Konto.",
+  "subscriptionReactivatedReminderBlob": "Vous avez atteint votre limite de pause et votre abonnement sera réactivé le {0}. Si vous souhaitez consulter ou gérer votre abonnement, rendez-vous sur <a href=\"https://{1}/myaccount\">Mon compte</a>.",
```

Comment:
plase arrange the label with alphabetical order

Reply:
Done